api/db/migrations/20210920153452_add-column-focusedout-to-answers.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'answers';
+const COLUMN_NAME = 'focusedOut';
```

Comment:
```suggestion
const COLUMN_NAME = 'isFocusedOut';
```

Reply:
Je propose de garder focusedOut pour : 
- la cohérence de lecture avec timedout déjà présent
- pour être aligné entre la bdd et le modèle.
Si on part sur isFocusedOut, je pense qu'il faudrait le renommer sur toute la grappe api alors.
Qu'en penses-tu ?

Reply:
@MelanieMEB je n'ai pas vraiment d'avis sur ce `is` en revanche attention, il n'y a pas de colonne `timeout` mais une colonne `timeout` qui n'est pas booléenne donc ça ne représente pas à mon sens un précédent pour décider si on met `is` devant les colonnes booléennes…

Reply:
J'ai l'impression que dans l'ensemble, on respecte bien le fait d'ajouter un `is`.
Je suis d'accord pour renommer côté API dans ce cas pour rester iso.

Reply:
J'ai remis tout à isFocusedOut